PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
-from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
-from Configuration.Eras.Modifier_run2_nanoAOD_94X2016_cff import run2_nanoAOD_94X2016
+from Configuration.Eras.Modifier_run2_jme_2016_cff import run2_jme_2016
```

Comment:
These two modifiers are imported, but never used here: are they supposed to stay here "just in case"?

Reply:
Yes, for just in case. But I can remove it if needed.

Reply:
Not requested. at least not by me.
It was just to verify that nothing was forgotten